firstvoices/backend/models/media.py

```diff
@@ -24,8 +21,27 @@
 
 from .base import AudienceMixin, BaseModel, BaseSiteContentModel
 from .constants import MAX_DESCRIPTION_LENGTH, MAX_FILEFIELD_LENGTH
+from .files import File, FileBase, file_directory_path
 from .validators import validate_no_duplicate_urls
 
+SUPPORTED_FILETYPES = {
+    # octet-stream is fallback for weird mp3 files; see fw-4829
+    "audio": [
+        "audio/wave",
+        "audio/wav",
+        "audio/x-wav",
+        "audio/x-pn-wav",
+        "audio/vnd.wav",
+        "audio/mpeg",
+        "audio/mp3",
+        "audio/mpeg3",
+        "audio/x-mpeg-3",
+        "application/octet-stream",
+    ],
+    "image": ["image/jpeg", "image/gif", "image/png", "image/tiff"],
+    "video": ["video/mp4", "video/quicktime"],
+}
+
```

Comment:
If we added an alias here, would it avoid the need to edit the migrations? E.g.,

`def media_directory_path = file_directory_path`

Reply:
Nice, this worked. I added an alias and migrations can now remain unedited. Thank you for the suggestion.